test/SymbolGraph/Symbols/SkipsPublicUnderscore.swift

```diff
@@ -7,6 +7,7 @@ public protocol PublicProtocol {}
 
 // CHECK-NOT: precise:{{.*}}_ProtocolShouldntAppear
 // CHECK-NOT: precise:{{.*}}PublicProtocol
+@_show_in_interface
 public protocol _ProtocolShouldntAppear {}
```

Comment:
Should we add a `CHECK-NOT` to make sure `_show_in_interface` doesn't appear in the JSON?

Reply:
I've got such a check in other commits. This test should check whether decls are present or not.